featurebyte/service/feature.py

```diff
@@ -87,12 +90,19 @@ async def prepare_feature_model(
             node=node,
             sanitize_for_definition=sanitize_for_definition,
         )
+        # extract entity relationships
+        relationships_info = await self.entity_relationship_extractor_service.extract(
+            entity_ids=QueryGraph(**prepared_graph.dict(by_alias=True)).get_entity_ids(
+                node_name=prepared_node_name
+            )
+        )
         return FeatureModel(
             **{
                 **data_dict,
                 "graph": prepared_graph,
                 "node_name": prepared_node_name,
                 "readiness": FeatureReadiness.DRAFT,
+                "relationships_info": relationships_info,
```

Comment:
This PR only adds `relationships_info` into feature model. This attribute is not used at the moment. Will be used later once we update feature list logic.